src/services/helpers/service.js

```diff
@@ -1,7 +1,11 @@
-const nodemailer = require('nodemailer');
+const request = require('request-promise-native');
+const { Unavailable } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/errors'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4774270)